jetty-deploy/pom.xml

```diff
@@ -59,5 +59,9 @@
       <artifactId>jetty-test-helper</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+    </dependency>
```

Comment:
Oops, wrong missing `<scope>test</scope>`!

Submitted a PR to correct at https://github.com/jetty/jetty.project/pull/10813